test/cypress/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "cypress-studio",
+  "name": "cypress-altinn-app-frontend",
   "version": "1.0.0",
-  "description": "studio tests",
+  "description": "altinn-app-frontend tests",
   "main": "index.js",
   "config": {
     "localtestPath": "C:/Repos/altinn-studio/src/development",
```

Comment:
Whats the significance of these paths? Where are they used?

Reply:
these paths are used in the scripts [here](https://github.com/Altinn/app-frontend-react/blob/test/remove-studio-tests/test/cypress/package.json#L25), one can change the path as described [here ](https://github.com/Altinn/app-frontend-react/blob/test/remove-studio-tests/test/cypress/package.json#L25) based on the path they have the repo.